lib/tasks/init-client.js

```diff
@@ -15,5 +16,15 @@ export default function initClient (opts) {
     ...defaultOpts,
     ...opts
   }
-  return createClient(config)
+  if (useCda) {
+    const cdaConfig = {
+      space: config.spaceId,
+      accessToken: config.cdaAccessToken,
+      environment: config.environmentId,
+      host: config.includeDrafts ? 'preview.contentful.com' : 'cdn.contentful.com',
```

Comment:
This is sort of tricky-- if the whole point of adding the option to fetch data from the CDA is to get the most recently published version of something, why would you want to include drafts? 

And then if you send --delivery-token and --include-drafts, what you're getting is preview data?

On one hand it's clever, but I'm worried about bugs coming from user error. Please make this super explicit in the README.

Reply:
That's a good point.

We're using the `includeDrafts` feature inside our preview server that allows users to preview drafts before they are published, but in production we want to fetch only the published data from CDA

Looking at this now, this seems to prevent us from viewing unpublished changes for published entries inside our preview server. I'll look into this a bit more

Reply:
After a quick review, this seems to work as expected. Using the preview API, you still get the unpublished changes for published entries, which is correct for this use case. I'll explain this in the README.